diff --git a/node_modules/date-fns/parse.js b/node_modules/date-fns/parse.js
index 5cbfcace9db1bbc3b48fe53f940abea2c0ee770f..b5f3e52bcaf92bd7a8a29ced3f686b621e3af1bc 100644
--- a/node_modules/date-fns/parse.js
+++ b/node_modules/date-fns/parse.js
@@ -7,7 +7,7 @@ var _index8 = _interopRequireDefault(require("./locale/en-US/index.js"));
 var _index9 = _interopRequireDefault(require("./_lib/toInteger/index.js"));
 var _index10 = _interopRequireDefault(require("./_lib/requiredArgs/index.js"));
 var _index11 = require("./constants/index.js");
-var _index12 = require("./_lib/format/longFormatters/index.js");
+var format_longFormatters = require("./_lib/format/longFormatters/index.js");
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
@@ -241,7 +241,7 @@ function parse(dirtyDateString, dirtyFormatString, dirtyReferenceDate, dirtyOpti
         // Long format notation
         var longFormatter = function longFormatter(pattern) {
           var dateTime = value.match(DATETIME_REGEX);
-          var date = !dateTime && value.match(DATE_REGEX);
+          var date = !dateTime && value.match(DATE_REGEX); 
           var time = value.match(TIME_REGEX);
           var result;
 
@@ -249,13 +249,13 @@ function parse(dirtyDateString, dirtyFormatString, dirtyReferenceDate, dirtyOpti
             var newFormat = 'YYYY-MM-DDTHH:mm:ss' + pattern;
             var dateTimeString = toDateTimeString(dateTime);
             result = parseString(dateTimeString, newFormat);
-          } else if (date && Object.prototype.hasOwnProperty.call(_index12, pattern)) {
+          } else if (date && Object.prototype.hasOwnProperty.call(format_longFormatters, pattern)) {
             var newFormat = 'YYYY-MM-DD' + pattern;
             var dateString = toDateString(date);
-            var dateStringWithPattern = dateString + (0, _index12[pattern])(null, {
+            var dateStringWithPattern = dateString + (0, format_longFormatters[pattern])(null, {
               width: 'short'
             });
-            result = parseString(dateStringWithPattern, newFormat + (0, _index12[pattern])(null, {
+            result = parseString(dateStringWithPattern, newFormat + (0, format_longFormatters[pattern])(null, {
               width: 'short'
             }));
 
</rewritten_file> 